kratos/tests/test_restart.py

```diff
@@ -304,4 +302,5 @@ def test_save_restart_process(self):
 
 
 if __name__ == '__main__':
+    KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(KratosMultiphysics.Logger.Severity.WARNING)
```

Comment:
As a general comment, even if it is very convenient to have tests produce no output when run in full suites, I normally find it useful for single test runs to have some output.

Reply:
I tend to agree, but the serializer gives a lot of output, which makes it hard to see really anything

I will think abt it